gnarly.rb

```diff
@@ -270,6 +276,19 @@ def setup_readme
   gsub_file "README.md", "__application_name__", app_name
 end
 
+def setup_i18n
```

Comment:
Do you think we should be more specific in the name of this method? Just thinking that even without this - your rails app is going to have i18n set up just fine. `configure_i18n`? `raise_on_missing_translations`?